Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/OthersContentPage.xaml

```diff
@@ -20,6 +20,9 @@
             <ViewCell Tapped="SignOut">
                 <Button Text="Sign Out" InputTransparent="true" Clicked="SignOut" />
             </ViewCell>
+            <ViewCell Tapped="SignInInformation">
+                <Button x:Name="SignInInformationButton" Text="User Not Authenticated" InputTransparent="true" Clicked="SignInInformation" />
```

Comment:
I prefer setting default value as "Unknown" until the first call to SignIn is made. Because user could actually be authenticated, we just don't know.